Gems/Atom/Feature/Common/Assets/Materials/materialpipeline_shader_structure.md

```diff
@@ -0,0 +1,233 @@
+### Glossary
```

Comment:
Thanks for writing this out. This should also go in the main document site. Please connect with sig-documentaiotn for help with this.